tasks/build-fits-site.yml

```diff
@@ -0,0 +1,83 @@
+- name: Build site for FITS microservice
+  blockinfile:
+    path: "/etc/apache2/conf-available/CrayFits.conf"
+    create: yes
+    block: |
+      Alias "/crayfits" "/var/www/html/CrayFits/public"
+      <Directory "/var/www/html/CrayFits/public">
+        FallbackResource /index.php
+        Require all granted
+        DirectoryIndex index.php
+        SetEnvIf Authorization "(.*)" HTTP_AUTHORIZATION=$1
+      </Directory>
+
+- name: Enable FITS site
+  file:
+    src: "/etc/apache2/conf-available/CrayFits.conf"
+    dest: "/etc/apache2/conf-enabled/CrayFits.conf"
+    state: link
+
+- name: Change owner on html dir
+  file:
+    path: /var/www/html
+    owner: www-data
+    group: www-data
+
+- name: Download islandora FITS module
+  composer:
+    command: require
+    arguments: islandora-rdm/islandora_fits
```

Comment:
I don't know the composer command, but I think this should specify a version so we can pin it.

Reply:
Due to my discovery in [this comment](https://github.com/Islandora-Devops/ansible-role-fits/pull/3/files#r315348993), this comment is irrelevant. The version is specified when installed through the `drupal_composer_dependencies` variable